### ISSUE TITLE
Add "Missing" indicator on the home page

### ISSUE THREAD

**Issue by pzwsk** [GFDRR/open-risk-data-dashboard#470]
Option 1
![image](https://user-images.githubusercontent.com/2378439/54704879-e776ff80-4b3b-11e9-95bc-1f569cf64f0b.png)

Option 2
![image](https://user-images.githubusercontent.com/2378439/54706766-f790de00-4b3f-11e9-8523-bcfdee30f6ec.png)

**Comment by vdeparday** [comment]
I kind of link the first one better because it is an indicator we can see progress and report on at the end of each year but any way we can add a bit more context or call for action ?  Maybe reword "still missing" and still use a sentence below to call for contribution to fill the misisng datasets ? Maybe we could even make that call a bit longer. Check your country, improve your country scores, add any missing datasets, etc... So that it is more obvious for people to contribute.

**Comment by vdeparday** [comment]
Re-upping this one in the priority. I really think we should have the missing on the homepage so that people start contributing, and we can do comms to getting it to 0 !
